.gitlab-ci.yml

```diff
@@ -1,11 +1,9 @@
-before_script:
-  - apt-get update
-
-after_script:
-  - gradle --version
-
 bullseye-jdk11:
   image: debian:bullseye-slim
+  before_script:
+    - apt-get update
+  after_script:
+    - gradle --version
   script:
     - apt-get -y install openjdk-11-jdk-headless gradle
```

Comment:
Since `before_script` isn't shared any more, we should move the `apt-get install` line to `before_script`.

Reply:
And after that, we since the `script` will have become the same, we could think about sharing that instead. I would like the restructure on the basis that it would better demonstrate that our product can be built with a simple `gradle build`, no matter what environment.

Reply:
> Since `before_script` isn't shared any more, we should move the `apt-get install` line to `before_script`.

Fixed and force-pushed.

Reply:
> And after that, since the `script` will have become the same, we could think about sharing that instead. I would like the restructure on the basis that it would better demonstrate that our product can be built with a simple `gradle build`, no matter what environment.

I assume you mean to do this in a separate PR.